libkineto/src/ActivityLoggerFactory.h

```diff
@@ -12,7 +12,9 @@
 #include <functional>
 #include <map>
 #include <string>
-
+#if defined(_MSC_VER)
```

Comment:
Can you please explain why `cctype` is needed, as Kineto can already be built fine with VC++?

Reply:
That is to enable `std::transform` and `std::tolower`. I'm not sure why it builds with VC++ before.

Reply:
`std::transform` is defined in algorithm, and `std::tolower` in indeed in `cctype`, but why does it need to be gated behind `#ifdef _MSC_VER`